debian/copyright

```diff
@@ -0,0 +1,33 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: llmnrd
+Source: https://github.com/tklauser/llmnrd
+
+Files: *
+Copyright: 2016 Tobias Klauser <tklauser@distanz.ch>
+License: GPL-2.0+
+
+Files: debian/*
+Copyright: 2016 Tobias Klauser <tklauser@distanz.ch>
+License: GPL-2.0+
+
+License: GPL-2.0+
```

Comment:
Check e.g. the netsniff-ng copyright (which is GPL-2 only too) for an example if you like: http://metadata.ftp-master.debian.org/changelogs/main/n/netsniff-ng/netsniff-ng_0.6.1-2_copyright